### PR TITLE
[GP-2] Support show/hide of Google / iCal export links

### PR DESCRIPTION
**Pull request by andrasguseo**
A user wrote on wordpress.org and I was curious if I could do this. So I did. :)
https://central.tri.be/issues/138300

[GP-2]

[GP-2]: https://moderntribe.atlassian.net/browse/GP-2

### CODE REVIEW

**Comment by andrasguseo** [comment]
Note, the date in the readme needs to be updated to the release date.

File: templates/shows-list.php [hunk 1]
```diff
@@ -38,17 +38,27 @@
...
-				<a class="gigpress-links-toggle" href="#calendar-links-<?php echo $showdata['id']; ?>">Add</a>
-				<div class="gigpress-calendar-links" id="calendar-links-<?php echo $showdata['id']; ?>">
-					<div class="gigpress-calendar-links-inner">
-						<span><?php echo $showdata['gcal']; ?></span>
-						<span><?php echo $showdata['ical']; ?></span>
-					</div>
-				</div>
-			</div>
-			<?php endif; ?>
+			if( $scope != 'past' ) :
```
Comment (sc0ttkclark) [inline templates/shows-list.php]:
```suggestion
			if ( $scope != 'past' ) :
```

File: admin/settings.php [hunk 2]
```diff
@@ -5,234 +5,322 @@ function gigpress_settings() {
...
+
+        <h1><?php _e( "Settings", "gigpress" ); ?></h1>
+
+        <form method="post" action="options.php">
+
+            <table class="gp-table form-table">
+                <tr>
+                    <th scope="row"><?php _e( "Full URL to your 'Upcoming Shows' page", "gigpress" ) ?>:</th>
+                    <td>
+                        <input type="text" size="48" name="gigpress_settings[shows_page]" value="<?php echo $gpo['shows_page']; ?>"/>
```
Comment (sc0ttkclark) [inline admin/settings.php]:
```suggestion
                        <input type="text" size="48" name="gigpress_settings[shows_page]" value="<?php echo esc_attr( $gpo['shows_page'] ); ?>"/>
```

File: admin/settings.php [hunk 3]
```diff
@@ -5,234 +5,322 @@ function gigpress_settings() {
...
+                    </td>
+                </tr>
+
+                <tr>
+                    <th scope="row"><?php _e( "User level required to use GigPress", "gigpress" ) ?>:</th>
+                    <td>
+                        <select name="gigpress_settings[user_level]">
+                            <option value="activate_plugins"<?php if ( $gpo['user_level'] == 'activate_plugins' ) {
+								echo( ' selected="selected"' );
+							} ?>><?php _e( "Administrator", "gigpress" ); ?></option>
```
Comment (sc0ttkclark) [inline admin/settings.php]:
```suggestion
							} ?>><?php esc_html_e( "Administrator", "gigpress" ); ?></option>
```

File: admin/settings.php [hunk 4]
```diff
@@ -5,234 +5,322 @@ function gigpress_settings() {
...
+                            <option value="edit_published_posts"<?php if ( $gpo['user_level'] == 'edit_published_posts' ) {
+								echo( ' selected="selected"' );
+							} ?>><?php _e( "Editor", "gigpress" ); ?></option>
```
Comment (sc0ttkclark) [inline admin/settings.php]:
```suggestion
							} ?>><?php esc_html_e( "Editor", "gigpress" ); ?></option>
```

File: admin/settings.php [hunk 5]
```diff
@@ -5,234 +5,322 @@ function gigpress_settings() {
...
+                            <option value="publish_posts"<?php if ( $gpo['user_level'] == 'publish_posts' ) {
+								echo( ' selected="selected"' );
+							} ?>><?php _e( "Author", "gigpress" ); ?></option>
+                            <option value="edit_posts"<?php if ( $gpo['user_level'] == 'edit_posts' ) {
+								echo( ' selected="selected"' );
+							} ?>><?php _e( "Contributor", "gigpress" ); ?></option>
```
Comment (sc0ttkclark) [inline admin/settings.php]:
```suggestion
							} ?>><?php esc_html_e( "Contributor", "gigpress" ); ?></option>
```

File: admin/settings.php [hunk 6]
```diff
@@ -5,234 +5,322 @@ function gigpress_settings() {
...
+                        <select name="gigpress_settings[user_level]">
+                            <option value="activate_plugins"<?php if ( $gpo['user_level'] == 'activate_plugins' ) {
+								echo( ' selected="selected"' );
+							} ?>><?php _e( "Administrator", "gigpress" ); ?></option>
+                            <option value="edit_published_posts"<?php if ( $gpo['user_level'] == 'edit_published_posts' ) {
+								echo( ' selected="selected"' );
+							} ?>><?php _e( "Editor", "gigpress" ); ?></option>
+                            <option value="publish_posts"<?php if ( $gpo['user_level'] == 'publish_posts' ) {
+								echo( ' selected="selected"' );
+							} ?>><?php _e( "Author", "gigpress" ); ?></option>
```
Comment (sc0ttkclark) [inline admin/settings.php]:
```suggestion
							} ?>><?php esc_html_e( "Author", "gigpress" ); ?></option>
```

File: readme.txt [hunk 7]
```diff
@@ -36,6 +36,10 @@ If you want to go beyond GigPress, we also have other plugins that could work gr
 
 == Changelog ==
 
+= 2.3.24 [TBD] =
+
+* Tweak - Added options to show / hide the Google Calendar and iCalender export links [138300]
```
Comment (sc0ttkclark) [inline readme.txt]:
```suggestion
* Tweak - Added options to show / hide the Google Calendar and iCalendar export links [GP-2]
```

**Pull request merged**